cmake/SetupThirdParty.cmake

```diff
@@ -26,7 +26,30 @@ endif()
 #------------------------------------------------------------------------------
 # axom
 #------------------------------------------------------------------------------
-if (DEFINED AXOM_DIR)
+if (TARGET axom)
+    # Case: Tribol included in project that also creates an axom target, no need to recreate axom
+    message(STATUS "Axom support is ON, using existing axom target")
+
+    # Check to make sure the components and dependencies of axom exist
+    # NOTE(chapman39@llnl.gov): Cannot simply install axom, since it is an imported library
+    set(_axom_comp ${AXOM_COMPONENTS_ENABLED};cli11;fmt;sol;lua;hdf5;sparsehash)
```

Comment:
Using this variable from Axom would clean this up and hopefully be more robust.

https://github.com/LLNL/axom/blob/982c6d3c00a607c8600119dd1db847fd5acff5e9/src/cmake/AxomConfig.cmake#L157C5-L157C25

Reply:
I agree with @white238  that we should use an export variable for the list of targets.

The names deps on this line aren't all required for axom, e.g. `lua`, `sol`, and `hdf5` are optional. I think `sparsehash` is also optional, but it's been a while since I last checked.

Reply:
Should the list be updated from `_axom_exported_targets` to something `AXOM_EXPORTED_TARGETS` now that it will be used outside that cmake section?

Reply:
Update: i ended up needing to use `axom_exported_targets` rather than `_axom_exported_targets`

Reply:
Also, i had to remove openmp from list, since it was included in another export set

From serac codevelop: `CMake Error: install(EXPORT "tribol-targets" ...) includes target "openmp" more than once in the export set.`

Reply:
That should be fixed when the new BLT is released but good job working around it.